recipes/svgwrite/all/conanfile.py

```diff
@@ -1,79 +1,91 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
 import os
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
 
+required_conan_version = ">=1.53.0"
 
 class SvgwriteConan(ConanFile):
     name = "svgwrite"
+    description = "a streaming svg library"
     license = "BSL-1.0"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://gitlab.com/dvd0101/svgwrite"
-    description = "SVGWrite - a streaming svg library"
-    topics = ("svg", "stream", "vector", "image")
-    settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = {"shared": False, "fPIC": True}
-    exports_sources = ("CMakeLists.txt", "patches/*")
-    requires = "span-lite/0.7.0", "fmt/6.1.2"
-    generators = "cmake", "cmake_find_package"
-    _cmake = None
+    topics = ("svg", "writer", "stream", "vector", "image")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _min_cppstd(self):
+        return 17
 
     @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "7",
+            "clang": "7",
+            "apple-clang": "10",
+        }
+
+    def export_sources(self):
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
-        compiler = str(self.settings.compiler)
-        compiler_version = tools.Version(self.settings.compiler.version)
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
 
-        if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, "17")
+    def layout(self):
+        cmake_layout(self, src_folder="src")
 
-        minimal_version = {
-            "Visual Studio": "16",
```

Comment:
It seems we lost the min VS version check?

Reply:
Thanks a lot.
I fixed it.